.github/workflows/nodejs.condo.ci.yml

```
@@ -620,6 +620,17 @@ jobs:
       - detect-changes
     if: ${{ needs.detect-changes.outputs.pos-integration == 'true' }}
     steps:
+      - name: Install mkcert for OIDC tests
+        run: |
+          sudo apt update && sudo apt install -y mkcert
```

actionlint (1.7.8)

could not parse as YAML: did not find expected key

(syntax-check)